### DISCUSSION TITLE
封面“本科毕业设计论文”字样存在中心偏移问题

### DISCUSSION THREAD

**Discussion by LS-King** [note286/xduts#11] [Q&A] [answered]
- 版本号：1.13.1.0
- 问题描述：在早期版本（如0.7.0.0）中，封面的校名、“本科毕业设计论文”、校徽、下方信息栏 四部分中心共线。但使用最新版本（1.13.1.0）编译测试后，“本科毕业设计论文”字样中心与其余三部分元素中心明显不共线，有所偏离。尚不确定该问题是在哪个版本引入的，仅测试了0.7.0.0和1.13.1.0两个版本。
- 补充：查阅了设计手册、撰写规范和封皮模板，未找到相关规范或依据。但基于常见封面设计规范和美观方面的考量，个人认为这确实是一个需要修正的问题。
- 图片展示：此处仅展示1.13.1.0版本中的编译后效果
![屏幕截图 2022-05-08 172642](https://user-images.githubusercontent.com/47248956/167290777-ba1abe47-8e1b-491b-921a-ecf00e431ef9.png)

**Comment by note286** [comment, marked as answer]
感谢提出，我一开始误解了，所以移动到了讨论区，在 b756e2e 已经修复。
